packages/core/src/anchor/anchor-processing-loop.ts

```diff
@@ -30,6 +34,11 @@ export class AnchorProcessingLoop {
     anchorStoreQueue: NamedTaskQueue
   ) {
     this.#anchorStoreQueue = anchorStoreQueue
+    this.#anchorPollingMetrics = new TimeableMetric(
+      SinceField.TIMESTAMP,
```

Comment:
@stbrody here is where it sets the "since" field ^^^^

Reply:
Yes I saw that, but seeing us set an enum that says we should be using a TIMESTAMP, did not make me immediately understand that whenever we call `record` we are comparing the current time to a timestamp taken from the document that we pass in, and that that document itself has a timestamp that was set, in a totally different part of the codebase, based on the timestamp of when the entry was persisted to the state store.  That's a lot of external context that is not available there at the call site.

Reply:
True!